src/components/popups-etc/help.tsx

```diff
@@ -1,13 +1,14 @@
 import { Signal, useSignal, useSignalEffect } from '@preact/signals'
-import { IoCaretDown, IoCaretUp } from 'react-icons/io5'
+import {IoCaretBack, IoCaretDown, IoCaretForward, IoCaretUp} from 'react-icons/io5'
```

Comment:
```suggestion
import { IoCaretBack, IoCaretDown, IoCaretForward, IoCaretUp } from 'react-icons/io5'
```

don't break the formatting!